app/code/core/Mage/Core/Model/Email.php

```diff
@@ -143,7 +144,25 @@ public function send()
         $mail->setFrom($this->getFromEmail(), $this->getFromName())
             ->addTo($this->getToEmail(), $this->getToName())
             ->setSubject($this->getSubject());
-        $mail->send();
+
+        Mage::dispatchEvent('email_send_before', array(
+            'mail'      => $mail,
+            'template'  => $this->getTemplate(),
+            'transport' => $transport,
+            'variables' => $this->getTemplateVars()
+        ));
+
+        if ($transport->getTransport()) {
```

Comment:
Bearing in mind line 139 does this `$transport = new Varien_Object();`, $transport will always be an empty object and therefore this line of code will never run.